src/App.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import AppRouter from "./components/AppRouter";
+
+const App = () => {
+  return (
+      <AppRouter />
```

Comment:
use the router directly here without creating the AppRouter component

```suggestion
<BrowserRouter>
            <Routes>
                <Route path='/login' element={<SignIn />} />
                <Route path='/shop' element={<Shop />} />
                <Route path='' element={<SignIn />} />
                <Route path='*' element={<SignIn />} />
            </Routes>
        </BrowserRouter>
```

Reply:
fixed